### PR TITLE
Updates to version numbers, austraits functions, DESCRIPTION

### PR DESCRIPTION
**Pull request by ehwenk**
Updates to:

- DESCRIPTION file to add more packages
- add versions for austraits_lite
- other small changes to hopefully match new `{austraits}` version

### CODE REVIEW

File: austraits_package.qmd [hunk 1]
```diff
@@ -12,7 +12,7 @@ library(dplyr)
 library(stringr)
 library(austraits)
 
-austraits <- austraits:::austraits_lite
+austraits <- austraits:::austraits_3.0.2_lite
```
Comment (yangsophieee) [inline austraits_package.qmd:15]: What's the reason we're using v3.0.2? Is it going to be updated later?

File: austraits_package.qmd [hunk 2]
```diff
@@ -12,7 +12,7 @@ library(dplyr)
...
 austraits$build_info$version <- "0.0.0.900"
```
Comment (yangsophieee) [inline austraits_package.qmd:16]: And this version too

**Review by dfalster — APPROVED** [review]

File: tutorial_dataset_1.qmd [hunk 3]
```diff
@@ -435,7 +435,7 @@ source("build.R")
 As a final step, build a report for the study
 
 ```{r, eval=FALSE}
-traits.build_database$build_info$version <- "4.0.0"  # a fix because the function was built around specific AusTraits versions
+traits.build_database$build_info$version <- "5.0.0"  # a fix because the function was built around specific AusTraits versions
```
Comment (yangsophieee) [inline tutorial_dataset_1.qmd:438]: What does `traits.build_database$build_info$version` evaluate to before rewriting to "5.0.0"? Will this be fixed eventually?

**Review by yangsophieee — COMMENTED** [review]
Just seeing if these comments will go through in an already merged branch

**Pull request merged**